curriculum/challenges/english/blocks/workshop-shape-manager/68ff495abf98da70d59e6f44.md

```diff
@@ -0,0 +1,331 @@
+---
+id: 68ff495abf98da70d59e6f44
+title: Step 6
+challengeType: 25
+dashedName: step-6
+---
+
+# --description--
+
+You need to select the HTML elements to work with. For this, you can create a function that gets an element and throws an error when the element is not found.
+
+Here's an example that shows how to create a function with types in Typescript:
+
+```ts
+function addNumbers(a: number, b: number): number {
+  return a + b;
+}
+```
+
+Create a `GetElements` function that takes a generic type `T` that extends `HTMLElement`, and a `selector` of type `string` as parameters. For now, leave the return type and the function block empty.
```

Comment:
It seems generics are taught in the lesson after this workshop. So this is something we should address.